docs/configuration.md

```diff
@@ -192,19 +192,20 @@ end up paying more then would probably have been necessary.
 
 ### Understand order_types
 
-The `order_types` configuration parameter contains a dict mapping order-types to
-market-types as well as stoploss on or off exchange type and stoploss on exchange
-update interval in seconds. This allows to buy using limit orders, sell using
+The `order_types` configuration parameter maps actions (`buy`, `sell`, `stoploss`) to order-types (`market`, `limit`, ...) as well as configures stoploss to be on the exchange and stoploss on exchange update interval in seconds.
```

Comment:
"and defines stoploss on exchange update interval."

(which is strange, btw (to have this parameter amongst order types in this dict)...)

Reply:
Yes it is ... maybe we shall think of moving it to a "regular" config spot in the future.